src/services/group.service.ts

```diff
@@ -26,4 +28,18 @@ export class GroupService {
   async deleteGroup(id: string) {
     return await Group.destroy({ where: { id } });
   }
+
+  async addUsersToGroup(userIds: string[], groupId: string) {
+    try {
+      return await sequelize.transaction(async (t) => {
+        const users = await User.findAll({ where: { id: { [Op.in]: userIds } }, transaction: t });
+        const group = await Group.findByPk(groupId, { transaction: t });
+
+        // How to manage m2m relations in a proper way?
```

Comment:
You can use mixins, example https://sequelize.org/api/v7/interfaces/belongstomanyaddassociationmixinoptions 
Also you can make some kind of scripts to automate the process, example https://github.com/ahmerb/sequelize-typescript-associations